externs/browser/w3c_speech.js

```diff
@@ -72,6 +72,7 @@ HTMLInputElement.prototype.onwebkitspeechchange;
 // W3C Web Speech API implemented in Chrome M23
 /**
  * @constructor
+ * @extends {EventTarget}
```

Comment:
Since we cannot make `EventTarget` a constructor at this time, perhaps you mean `@implements`?